AtEventDisplay/AtSidebar/AtSidebarIntMacro.h

```diff
@@ -0,0 +1,54 @@
+#ifndef ATSIDEBARINTMACRO_H
+#define ATSIDEBARINTMACRO_H
+
+#include "AtSidebarAddon.h"
+
+#include <Rtypes.h> // for THashConsistencyHolder, UInt_t, ClassDef...
+
+#include <GuiTypes.h> // for Pixel_t
+
+#include <functional>
+#include <string>
+#include <utility> // for move
+
+class TBuffer;
+class TClass;
+class TGWindow;
+class TMemberInspector;
+
+class AtSidebarIntMacro : public AtSidebarAddon {
+private:
+   using MacroFunction = std::function<void(int)>;
+   MacroFunction fFunction;
+
+   std::string fLabel;
+
+   int fMin{0};
+   int fMax{10};
+   int fStart{0};
```

Comment:
These are determined at creation time? Is there a use case where they would be specified separately or can they be added to the constructor and not stored as part of the state of the class?